requirements-docs.txt

```diff
@@ -7,8 +7,9 @@ gitpython # Required for parsing git info for generation of data-adapter docs
 grpcio-status
 mock==1.0.1 # read the docs pins
 myst-parser==0.18.1  # latest version of myst at this time
-pillow
 # required for all the plugins
+pandas-gbq
+pillow
```

Comment:
why are these needed?

Reply:
if we move the import in `pandas_extensions.py` I think we will no need `pandas-gbq` here.

Reply:
i'm not sure, it looks like it was previously in the .txt file, but one of the pre-commit hooks alphabetized it? 
i can move the `pandas-gbq` library import to `pandas_extensions.py`.

Reply:
🤦  lol. oh now I see it. Thanks